docs/UserGuide.md

```diff
@@ -36,213 +148,261 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 6. Refer to the [Features](#features) below for details of each command.
 
---------------------------------------------------------------------------------------------------------------------
+### 3.2 Graphical User Interface Layout:
+<a id="graphical-user-interface-layout"></a>
 
-## Features
+#### 3.2.1 User Interface Overview:
+<a id="user-interface-overview"></a>
 
-<box type="info" seamless>
+TO BE DONE
 
-**Notes about the command format:**<br>
+#### 3.2.2 Additional UI Components:
+<a id="additional-ui-components"></a>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+TO BE DONE
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+### 3.3 How to use TrackMate commands:
+<a id="how-to-use-track-mate-commands"></a>
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+TrackMate operates primarily through text-based commands. Before we explore the specific commands in detail in the 
+Commands section, let’s familiarize ourselves with the basic components and format of a command.
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+#### 3.3.1 Parameter Prefixes:
+<a id="parameter-prefixes"></a>
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+In TrackMate, a parameter prefix acts as a delimiter for specifying different types of parameters in commands. 
+Here's a reference table for common parameter prefixes and their corresponding parameters:
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
-  </box>
+TO BE DONE
 
-### Viewing help : `help`
+#### 3.3.2 Parameters:
```

Comment:
I like how you've clearly presented the parameters! It makes it easy to understand and reference them!